documentation/auto-discovery/topics/advanced-discovery-topics.rst

```diff
@@ -0,0 +1,100 @@
+.. include:: /auto-discovery/media.rst
+.. _Advanced-discovery-topics-doc:
+
+*************************
+Advanced discovery topics
+*************************
+
+====================
+Obtaining SNMP walks
+====================
+
+Hyperview uses SNMP walks to enhance device definitions, to model and support devices that are discoverable with the SNMP protocol. The **snmpwalk file** is used to simulate the device and to test definitions.
+
+It is recommended to install the applicable net-snmp package alongside the (Linux) Data Collector software. If that is not possible it can be installed on any machine that has a network line of sight to the target device.
+
+Linux
+-----
+
+The `net-snmp <http://www.net-snmp.org/>`_ utilities are available on all supported Linux distributions.
+
+On Debian-based distributions:
+
+.. code::
+
+    sudo apt install snmp
+
+On RedHat-based distributions:
+
+.. code::
+
+    sudo dnf install net-snmp-utils
+
+Windows
+-------
+
+There are multiple options.
+
+1. Use net-snmp using a UNIX environment simulator such as `Cygwin <https://www.cygwin.com/>`_ or `WSL <https://docs.microsoft.com/en-us/windows/wsl/>`_.
+
+2. Use a Linux Virtual Machine
+
+macOS
+-----
+
+You can install net-snmp on macOS using `Homebrew <https://brew.sh/>`_
+
+.. code::
+
+    brew install net-snmp
+
+Once the application is installed, the **snmpwalk** command can be used to obtain a full walk of a device. For example:
```

Comment:
1. Since you used "for example".  I don't know if that makes it clear that we want it with ObentU used.  I think you should emphasize this.  Also there is a -o option for output, I believe.
2. Note there is old documentation we used to send to customers that you may want to reference for helpful text.
3. At least note that some large network devices may take awhile. please.

Reply:
Noted. I made some updates based on this feedback. Thanks!